src/frontend/src/sass/_default.scss

```diff
@@ -86,21 +87,23 @@ h4 {
 .page__controls {
   display: flex;
   justify-content: space-between;
-  min-width: 70%;
+  min-width: 100%;
+  padding-right: 12rem;
 }
 
 .component__wrapper {
   display: flex;
   background-color: $bg-white;
   flex-direction: column;
   align-items: center;
-  min-width: 70%;
+  min-width: 90%;
   box-shadow: 0 17px 29px rgba(220, 226, 238, 0.6);
   border-radius: 4px;
   padding-block: 5em;
   padding-inline: 7em;
   margin-block: 2em;
   margin-inline: 2em;
+  margin-right: 12rem;
```

Comment:
There is already an inline margin right above this property. Ideally, we should be positioning things using flex or grid properties in the parent element.